ifStatement.js

```diff
@@ -59,6 +73,13 @@ Steps:
     - If the user is younger than 18, the program should calculate how many years are left until they turn 18 and print the message "You will be eligible to vote in X years", where X is the number of years left.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let userage=18
+if (userage>=18)
+if(confirm("press" )) console.log(`you press ok `);
```

Comment:
The nested if-else statements are not properly structured. You need to use curly braces {} to define the block of code for each if-else condition.

The confirmation message within the confirm() function is missing. It should provide some context for the user to make a decision!